internal/static/i18n/de.yaml

```
@@ -113,4 +113,171 @@ Errors:
   Token:
     NotFound: Token konnte nicht gefunden werden
   UserSession:
-    NotFound: UserSession konnte nicht gefunden werden
\ No newline at end of file
+    NotFound: UserSession konnte nicht gefunden werden
+EventTypes:
+  user:
+    added: Benutzer hinzugefügt
+    selfregistered: Benutzer hat sich selbst registriert
+    initialization: 
+      code:
+        added: Initialisierungscode hinzugefügt
```

```suggestion
        added: Initialisierungscode generiert
```

i think `generiert` is the better word

---

I agree